uui/components/datePickers/__tests__/__snapshots__/RangeDatePickerBody.test.tsx.snap

```diff
@@ -45,575 +45,571 @@ exports[`RangeDatePickerBody should be rendered correctly 1`] = `
           }
         >
           <div
-            className="uui-datepicker-container fromPicker"
```

Comment:
Basically the same diff here. We should first decide what to do [here](https://github.com/epam/UUI/pull/1943/files#r1490747200)